attunement.lic

```diff
@@ -9,21 +9,41 @@ class Attunement
   include DRCA
   include DRCT
 
+  LUNAR_PERCEIVE_COMMANDS = ['mana', 'moons', 'planets']
+
   def initialize
     settings = get_settings
     @stationary_skills_only = settings.crossing_training_stationary_skills_only
     @hometown = settings.hometown
     @attunement_rooms = settings.attunement_rooms
-    @start_timer = Time.now
+    @exp_timer = settings.exp_timers['Attunement'] || 90
```

Comment:
This could make a good validator, thoughts?

Reply:
Yes, I'll hold off on writing the validator until we come to consensus on which settings will be used

Reply:
Validations updated per https://github.com/rpherbig/dr-scripts/pull/4486/commits/0d711f7579f8d0c5a9efcacb9a9f7cca8b36e8e5